### PR TITLE
Added presentation based on the analysis

### PR DESCRIPTION
**Pull request by varsim91**
Added presentation based on the analysis

### CODE REVIEW

File: Group_Facilitator_Name.ipynb [hunk 1]
```diff
@@ -1083,7 +1080,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "quantitative_cols=['loan_amnt','dti','funded_amnt','installment','funded_amnt_inv','int_rate','annual_inc','grade']\n",
```
Comment (venkateshgaddam) [inline Group_Facilitator_Name.ipynb]: why did you remove this column?

File: Group_Facilitator_Name.ipynb [hunk 2]
```diff
@@ -425,7 +425,7 @@
     "# Check for possible columns with outliers\n",
     "#\n",
     "numerical_cols = [\n",
-    "    'loan_amnt', 'int_rate', 'installment', 'annual_inc', 'dti', 'revol_util', 'open_acc', 'total_acc'\n",
```
Comment (venkateshgaddam) [inline Group_Facilitator_Name.ipynb]: Is this Column causing issue?

File: Group_Facilitator_Name.ipynb (hunk 1 above)
Comment (varsim91) [reply Group_Facilitator_Name.ipynb]: We cannot do sns.boxplot(x='grade', data=data, ax=p[1]) as grade is a categorical variable. While calling univariate_analysis(), we should not pass 'grade`

File: Group_Facilitator_Name.ipynb (hunk 2 above)
Comment (varsim91) [reply Group_Facilitator_Name.ipynb]: 'revol_util' has NaN values which was causing the crash
